src/common/printLoclaStorage.js

```diff
@@ -0,0 +1,32 @@
+import kIcon from '../images/3dot.png';
+import bIcon from '../images/bin.png';
+
+const binIcon = new Image();
+binIcon.src = bIcon;
+binIcon.classList.add('icon');
+
+const kebabIcon = new Image();
+kebabIcon.src = kIcon;
+kebabIcon.classList.add('icon');
+const printLocalStorage = () => {
+  if (localStorage.getItem('toDolist') != null) {
+    const storedList = JSON.parse(localStorage.getItem('toDolist'));
+    /* eslint-disable */
+    for (let i = 0; i < storedList.length; i++) {
+      /* eslint-enable */
+      storedList[i].index = i + 1;
+    }
+    /* eslint-disable */
+        for (let i = 0; i < storedList.length; i++) {
+    /* eslint-enable */
+      const div = document.createElement('div');
```

Comment:
- [ ] Good job fixing some of the linter errors as requested by the previous reviewer but you seem to have missed some, kindly fix all linter errors instead of disabling them. As a reminder, also check in other `.js` files to confirm that all linters are fixed and not disabled.

Reply:
resolved.